pnpm-lock.yaml

```diff
@@ -43,7 +45,7 @@ importers:
         version: 1.3.2
       '@rollup/plugin-typescript':
         specifier: ^11.1.5
-        version: 11.1.5(rollup@4.6.1)(typescript@5.3.2)
+        version: 11.1.5(rollup@4.6.1)(typescript@4.9.5)
```

Comment:
I think we need to revert this change. It introduced by running ./scripts/test.sh locally, and the reason is in the template: https://github.com/Lightprotocol/light-protocol/blob/main/psp-template/psp-template/package.json#L26

Reply:
OK. Any ideas how can I revert safely just that one change (downgrade of typescript) while keeping other relevant parts of the lock file?

Reply:
you could try to upgrade the version in the psp-template